### PR TITLE
fix: removed hard coded strings in java files and replaced them with strings

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes: #819
Removed all the hardcoded strings and converted them intro strings.xml strings
- [.] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [.] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [.] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/fragment/UpiPinFragment.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         if (b != null) {
             step = b.getInt(Constants.STEP, 0);
             upiPin = b.getString(Constants.UPI_PIN, null);
-            mTvTitle.setText("Re-enter UPI PIN");
+            mTvTitle.setText(R.string.re_enter_pin);
```
Comment (jawidMuhammadi) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/fragment/UpiPinFragment.java]: Are you sure it is working as expected and displaying the correct text? As I see you are setting the id of the string as the text, I think inside the java code we have to do something like this `getString(R.string.re_enter_pin);`

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/fragment/UpiPinFragment.java [hunk 2]
```diff
@@ -98,19 +98,19 @@ public void okayClicked() {
                     if (upiPin.equals(mPeUpiPin.getText().toString())) {
                         ((SetupUpiPinActivity) getActivity()).upiPinConfirmed(upiPin);
                     } else {
-                        showToast("UPI PIN mismatch");
+                        showToast(R.string.upi_pin_mismatch);
```
Comment (jawidMuhammadi) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/fragment/UpiPinFragment.java]: Same here and in all your other string changes.

**Comment by shiv07tiwari** [comment]
@Bhavnaharitsa I agree with @jawidMuhammadi . Please fix it and also look into the Travis Build issue.

**Comment by Bhavnaharitsa** [comment]
@shiv07tiwari Yes, will do the neeful

**Pull request closed without merging**